### DISCUSSION TITLE
What library/API is used for generating the default avatar for profiles?

### DISCUSSION THREAD

**Discussion by nicejs-is-cool** [community/community#29735] [Other Feature Feedback, Questions, & Ideas] [answered]
I heard of a library that could do Github's style of avatars and also Google's style.
But I can't seem to find it anywhere.

**Comment by KaweMaximo** [comment]
an avatar from github profile or github rewards?

**Reply by nicejs-is-cool** [reply]
GitHub profile.

**Reply by KaweMaximo** [reply]
JavaScript(Beginner) ?

**Comment by davidegiacometti** [comment, marked as answer]
They are called identicons. You may able to find a library for your desired language searching this term.

_An Identicon is a visual representation of a hash value, usually of an IP address, that serves to identify a user of a computer system as a form of avatar while protecting the user's privacy. The original Identicon was a 9-block graphic, and the representation has been extended to other graphic forms by third parties._

- https://github.blog/2013-08-14-identicons
- https://en.wikipedia.org/wiki/Identicon
